package.json

```diff
@@ -29,7 +29,7 @@
     "lodash": "3.10.1",
     "node-ansiparser": "2.0.2",
     "octicons": "2.4.1",
-    "pty.js": "0.3.0",
+    "ptyw.js": "^0.3.2",
```

Comment:
@shockone If I point to http://github.com/iiegor/pty.js, the git submodule of the repo doesn't initialize, so node-gyp rebuild fails :confused: